Gemfile

```diff
@@ -3,7 +3,7 @@ source 'http://rubygems.org'
 # Add dependencies required to use your gem here.
 gem 'rails', '~> 4.2'
 
-gem 'maestrano-rails', '~> 1.0.0'
+gem 'maestrano-rails', '~> 1.0.3'
```

Comment:
Not related to this PR, but to the general structure

For gems/engine, you shouldn't use the `Gemfile` but the gemspec.
The `Gemfile` should only contain the `gemspec` line.

See:
https://github.com/plataformatec/devise/blob/master/Gemfile
https://github.com/maestrano/mno-enterprise/blob/master/core/Gemfile

Btw, looking at the gemspec, it should be split between dev and runtime dependencies!